src/list-of-meals/mealsManager.ts

```diff
@@ -78,6 +78,20 @@ export class MealsManagerBL {
     }, []);
   }
 
+  checkEaten(meals: IMealBL[]) {
+    const isAllEaten = meals.every((value) => {
+      if (value.eaten === true) {
+        return true;
+      }
+    });
+
+    if (isAllEaten) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
could be simpler: 
`return isAllEaten`

which basically translates into either `return true` or `return false`

Reply:
rewrote